templates/endpoints_v2/README.ipynb

```diff
@@ -207,7 +252,7 @@
     "tags": []
    },
    "source": [
-    "## More guides\n",
+    "## Final workds on RayLLM\n",
```

Comment:
```suggestion
    "## What is RayLLM?\n",
```

Reply:
Can we add a sentence that clarifies that RayLLM wraps existing engines (vLLM, TRT) and adds scaling functionalities through ray to them and allowing you to productionize those optimizations ?